client/influxdb_test.go

```diff
@@ -14,6 +14,76 @@ import (
 	"github.com/influxdb/influxdb/client"
 )
 
+func BenchmarkUnmarshalJSON2Tags(b *testing.B) {
+	var bp client.BatchPoints
+	data := []byte(`
+{
+    "database": "foo",
+    "retentionPolicy": "bar",
+    "points": [
+        {
+            "name": "cpu",
+            "tags": {
+                "host": "server01",
+                "region": "us-east1"
+            },
+            "time": 14244733039069373,
+            "precision": "n",
+            "values": {
```

Comment:
is it `"fields"` or `"values"`?  it used to be fields and i'm not sure i can find where that has changed if it has.